test/unit/spec/LoginRouter_spec.js

```diff
@@ -174,7 +174,7 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, Logger, OktaAuth, Util, Exp
       expect(Logger.warn).toHaveBeenCalledWith('Field not defined in schema', arg1);
     }
 
-    it('throws a ConfigError if unknown option is passed as a widget param', function () {
+    it('logs a ConfigError error if unknown option is passed as a widget param', function () {
       spyOn(Logger, 'warn');
```

Comment:
Fun fact: We're **not** actually logging here.

There is a [_known_](https://oktainc.atlassian.net/browse/OKTA-170705) issue when instantiating the `BaseRouter`, where we're logging `Field not defined in schema: ${var}` for every widget param (`baseUrl`, `assets`, etc.).

Once that is resolved, we need to update these tests.

Reply:
They will not. The warning `Field not defined in schema` will only be visible when `window.okta && window.okta.debug` are defined, as it is still using Courage's `Logger` class.

With the new additions in this PR, any `Logger.X` method called within the Widget's source will now emit a console notification. We have only two usages of that today:

- [Use "scopes" instead of "scope"](https://github.com/okta/okta-signin-widget/blob/d168cf84402b503b17a7b4b1e967a6bf02ed03da/src/models/Settings.js#L443)
- [Use "i18n" instead of "labels" and "country"](https://github.com/okta/okta-signin-widget/blob/d168cf84402b503b17a7b4b1e967a6bf02ed03da/src/models/Settings.js#L449)

Reply:
got it, thanks for the explanation. model validation still uses Courage's Logger, makes sense now 👍